### ISSUE TITLE
No post when the chip is formatted with the latest modxo version

### ISSUE THREAD

**Issue by alejoortega03** [shalxmva/modxo#28]
Hello, i appreciate what you are doing with the OG Xbox, since it's more difficult to get our hands with a Aladdin chip, using the raspberry pi pico is a easier and cheaper way to hardmod a xbox, at least from my part of the world.

I was trying to install modxo on my 1.6 version xbox, guided by your diagram and steps, without cutting the pin 6 on the xyclops chip, and after verify the conecctions of the LPC cables and the chip cables itself, and formatting the pi pico with the latest version, my xbox wont boot on my tv and it shows a error code FRAG 50% red and green, the LED on the pi pico lights up but my xbox didn't post, sometimes it gives me the error code that try to boot 3 times, then FRAG. The only way i could make it boot on my screen was erasing the files from the pi pico with the flash_nuke.uf2 file. 

When i did that the LED on the pi pico didn't light up and my xbox booted as usual without any error code on screen or on the power button light.

What could have been the issue? I don't want to cut the pin 6 on the xyclops until someone confirm me that's the problem.

I think i dont have a replica of the raspberry pi pico, but i'm trying to figure that out.

Any help its useful, at least i know i didnt break my xbox. Thank you and reetings from Colombia.

**Comment by antoxa2584x** [comment]
Try another pico board.
No cut require on the pin 6, just lPC rebuild
Also, did you flash bios to the pico board after modxo flash?

**Comment by rdmrocha** [comment]
Can confirm that I also have an LPC rebuilt v1.6, no trace cut needed. Follow the instructions on the main github page: after flash_nuke, drag modxo uf2 to the drive and let it unplug itself. Reset+boot so the drive comes up again and drop your bios uf2 and let it unplug itself once flashed. I currently have cerbios 2.4.1 and it's the first time I see my xbox going from poweron to dashboard in ~12s.

If it's still failing and you're positive that your LPC rebuild is fine, together with all rp2040 and resistors, you may have stumbled upon an incompatible pico

**Comment by antoxa2584x** [comment]
@rdmrocha did 2.4.1 boot faster then 2.4.0?

**Comment by rdmrocha** [comment]
On my 1.6, yes. Bear in mind that I don't have any boot animation. But it still went down a couple of seconds.

**Comment by alejoortega03** [comment]
@rdmrocha how can i check if my pi pico is a generic one?

**Comment by alejoortega03** [comment]
@rdmrocha i'm using the green large one, do you recomend using the tiny one?

**Comment by rdmrocha** [comment]
> @rdmrocha i'm using the green large one, do you recomend using the tiny one? Or the Zero one with the usb C port
currently using some cheap aliexpress ones and they behave fine
![image](https://github.com/user-attachments/assets/eef43145-dfef-4564-94e8-3b57737327e9)

**Comment by antoxa2584x** [comment]
@rdmrocha I got issue with bios on one like this

**Comment by rdmrocha** [comment]
Must be hit or miss, I guess. I've been lucky with my batches

**Comment by alejoortega03** [comment]
My only chance is to get a diferent chip and check if the compatibility its the problem

**Comment by catman85** [comment]
I have the same issue with @alejoortega03. I have tested my setup extensively with a polymeter, and I can't find any issue. I am using an original rapsberry pi pico. I am running a 1.4 xbox. Could someone please share a uf2 file that is known to work? Preferably Cerbios udma2
![Uploading IMG_20240720_042136_812.jpg…]()

**Comment by catman85** [comment]
Update. I got my modxo to work with the latest version! I am not sure what the problem was. But here is what I did:
1. Since I was flashing the 2.4.1 Cerbios bios binary I updated my C:/cerbios.ini, since from this version the UDMA option is now dynamically configured.
2. I masked the chip with tape to protect it from shorting.
3. Originally the chip was sitting around the clock capacitor area (wires unecessarily too long) and the disc drive was applying pressure to it. I moved it closer to the LPC port.

Congrats on the project btw!
